README.md

```diff
@@ -1 +1,158 @@
-https://hub.docker.com/r/microsoft/oms/
+# Trying the container solution pack for Microsoft Operations Management Suite
```

Comment:
Note that I didn't check this README.md file for correctness. I'll leave that up to you.